cdap-ui/app/cdap/components/HeaderActions/HeaderActions.less

```diff
@@ -41,16 +41,20 @@
       }
       .plus-button {
         color: @plus-button-color;
+        position: relative;
         &:after {
           content: 'NEW';
           position: absolute;
           background: #ff6600;
-          font-size: 9px;
+          font-size: 8px;
           color: white;
           border-radius: 4px;
-          top: 11px;
-          padding: 2px;
+          top: -4px;
+          left: 10px;
+          padding: 1px 2px;
           font-weight: bold;
+          width: 23px;
+          height: 11px;
```

Comment:
The spec required it to be of specific height and width. I could remove it and it will be 1px less on both width and height. Let me see if I can remove the width and height